docs/node-operators/get-started/setup-datanode.md

```diff
@@ -73,8 +73,12 @@ Where:
 - `database_password` is the password you want to use to connect to the database.
 - `database_name` is the name of the database you want to use for storing the data.
 - `localdb_port` is the port you want to use to connect to the database on your local machine. (5432 is the default port for Postgresql database server and may not be available if you already have a postgresql database server running on your machine and want to use Docker for testing).
-- `/host_path/to/snapshotsCopyTo` is the path on your host machine where you want to store the snapshots that are generated by data node. Snapshots allow you to restore the database to a previous state.
-- `/host_path/to/snapshotsCopyFrom` is the path on your host machine where you want to store the snapshots that are retrieved from IPFS and can be used to rebuild a data node database from another data node.
+- `/host_path/to/snapshotsCopyTo` is the path on your host machine where you want to store the snapshots that are generated by data node. Snapshots allow you to restore the database to a previous state. By default, this folder will be in the `state/data-node/networkhistory/snapshotsCopyTo` folder under the `--home` folder you set for data node. [See Data Node Configuration](#data-node-configuration)
+- `/host_path/to/snapshotsCopyFrom` is the path on your host machine where you want to store the snapshots that are retrieved from IPFS and can be used to rebuild a data node database from another data node. By default, this folder will be in the `state/data-node/networkhistory/snapshotsCopyFrom` folder under the `--home` folder you set for data node. [See Data Node Configuration](#data-node-configuration)
+
+:::note Snapshot paths
+If you intend to use the Docker version of TimescaleDB and you do not mount the `snapshotsCopyTo` and `snapshotsCopyFrom` paths to the TimescaleDB container, Data Node will panic when trying to start from network history because TimescaleDB is not able to access the network history files Data Node retrieves and tries to import. The mounted path in the container must match the path of the `snapshotsCopyTo` and `snapshotsCopyFrom` paths on the Docker host.
+:::
```

Comment:
```suggestion
:::note Snapshot paths
If you intend to use the Docker version of TimescaleDB, the mounted path in the container must match the path of the `snapshotsCopyTo` and `snapshotsCopyFrom` paths on the Docker host. If you do not mount the `snapshotsCopyTo` and `snapshotsCopyFrom` paths to the TimescaleDB container, data node will panic when trying to start from network history because TimescaleDB is not able to access the network history files that data node retrieves and tries to import. 
:::
```